zirgen/compiler/picus/test/back.zir

```diff
@@ -0,0 +1,13 @@
+// RUN: zirgen %s --emit=picus | FileCheck %s
+
+// CHECK: (prime-number 2013265921)
+// CHECK: (begin-module Count)
+// CHECK: (input x0)
```

Comment:
Looking at the full output for this test, `x0` corresponds to the argument `first: Val` rather than the back. I think we should make a filecheck assertion about all of the inputs and signals corresponding to the back (`x1_*`).